apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	rules: {
+		// These components render in a Gutenberg environment and should
+		// conform to those naming conventions instead of Calypso's.
+		'wpcalypso/jsx-classname-namespace': 0,
```

Comment:
should this be applied to all of the FSE plugin?

Reply:
It probably should be. I didn't want to force folks to do a CSS refactor before they committed, but actually I've got that the wrong way round: this _removes_ the error.

So yeah it'd probably be safe to do that. I don't see any modules in the folder structure that look like they're _not_ gutenberg specific.

Reply:
Moved this rule to the root of FSE